### ISSUE TITLE
Extension causes images to render weirdly

### ISSUE THREAD

**Issue by ilyanep** [schmamps/angola-maldives#4]
I haven't been able to figure out why with some cursory messing around in developer tools, but for some reason with the extension enabled, the text in the comic renders weirdly.

With extension:
<img width="580" alt="dcwithextension" src="https://user-images.githubusercontent.com/346344/92441705-88f47580-f163-11ea-953e-73e755f95eb0.png">

Without:
<img width="560" alt="dcwithoutextension" src="https://user-images.githubusercontent.com/346344/92441754-a1fd2680-f163-11ea-8fa9-1ce2632b18a8.png">

Edit: Not sure how easy that is to see from the screenshots, but I'd describe it as "less bold" with the extension enabled, which makes the comic kind of hard to read.

I'm not familiar with how the extensions work, so I don't know if this will ring any bells for you. I'm running Windows 10 in 3840x2160 resolution with 150% OS scaling, if that might be the cause.

**Comment by schmamps** [comment]
DId this start happening recently? I know that [Ryan made some graphic changes](https://twitter.com/ryanqnorth/status/1285203263026089990) several weeks ago.

Anyway, it is nice to know I have a user. I will look into this as best I can (without 4K or Windows) and push a fix that also clears the dependency warnings GitHub keeps sending me.

**Comment by ilyanep** [comment]
It's hard to say -- I only started using it after recent changes broke the plugin in the Chrome Web Store and I saw your comment on there. Based on the description from that tweet, I would not at all be surprised if some changes in an overlay are doing this, since it really does look like it's messing with the text by a pixel or two.

If there's any info I can pull from Chrome that you think might be useful, let me know!

**Comment by schmamps** [comment]
I can't reproduce this, but I wonder what happens if you:

1. open `build.safariextension\maldives.css`
1. go to line 10 or so
1. remove or disable the `image-rendering` style, e.g.

```css
center > table tr > td:nth-child(2) > img {
  background-repeat: no-repeat;
  transition: all 0ms ease;
  width: auto;
  /* image-rendering: pixelated; */
}
```

Without reading logs, I think this had something to do with Safari some years ago.

**Comment by schmamps** [comment]
And reload the extension, of course.

**Comment by ilyanep** [comment]
I'll try that exact step later today after work, but I have evidence that you're probably right about that being the culprit. I did the following steps:

1. Disable the extension
2. Reload the page
3. Inspect element on the image
4. I see that the `.comic` class has `image-rendering: pixelated;` but Chrome has it crossed out and is using `image-rendering: auto;` on the element (though I can't tell what's overriding this or why)
5. If I click the checkbox to use `image-rendering: pixelated;` I can reproduce the effect on the text.

**Comment by ilyanep** [comment]
Yup, modifying the CSS in the extension did it!

**Comment by schmamps** [comment]
Fixed in e578ffb… make that 58a7c36

**Comment by schmamps** [comment]
If you've got a second and care at all about the alternate views, grab the v2.1.x update. 

It's got this fix, more alternate views, and valid URLs for them!
